src/content/release-notes.mdx

```diff
@@ -5,6 +5,38 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.26.0
+
+11 November 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### Breaking Changes
+
+- Hostname Length Limit: Deploys will now fail if a service generates a hostname longer than 63 characters, providing an error message to guide users. Previously, the endpoint wouldn’t work, but the dev environment still deployed successfully. This change may affect environments that previously deployed without issue.
+- Helm Release Name Limit: We are limiting the length of Helm release names and allowing to override the default backend service name to make it shorter if needed. The defaultBackend.nameOverride allows to change the full name of the default backend service. In case you want to change it as part of an upgrade, it might have side effects in the installation, so please, follow this guide
+- Deploying Private Repositories now uses the Okteto backend as the SSH agent instead of mounting the local SSH agent into BuildKit. This change provides feature parity between remote and local deploys but may affect cases where repositories are cloned as part of a command defined in your Okteto Manifest.
+- Buildkit Persistence is now enabled by default. The default disk size is set to 100Gi and cache size is now specified as a ratio of the total disk size (default: 0.9). For example, with a 100Gi disk, 90% will be allocated to cache. If you had previously set `buildkit.persistence.cache setting` please adjust the new ratio accordingly, as that setting will not apply any longer
+
+### New Features
```

Comment:
We should document the changes to configure "Deploy on Remote" from the Admin UI

Reply:
@ifbyol Will this be done in time?

Reply:
Yes, this is already done and included in the upcoming release

Reply:
Ok thanks. I just moved the ticket status to done and will update here